module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -482,9 +482,33 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             $itemResponse = $this->makeRequest('GET', '/item-storage/items', $query);
             $itemBody = json_decode($itemResponse->getBody());
             $notesFormatter = function ($note) {
-                return $note->note ?? '';
+                return !$note->staffOnly && $note->note ? $note->note : '';
```

Comment:
Are `staffOnly` and `note` guaranteed to be set, or would it be safer to do something like this instead?

`return !($note->staffOnly ?? false) && !empty($note->note) ? $note->note : '';`

Reply:
Done